### PR TITLE
No data component for Appointments

### PR DESCRIPTION
**Pull request by Funmi7**
##  What does this PR do
Renders a component when there is no data for appointments

##  Description of task to be completed
Show a component when there are no appointments for a user.
Shows a button that a user can click and go to the marketplace and see list of coaches

##  How should this manually be tested
npm start the application, clear the incoming booked appointments.

##  What are the relevant Trello board stories
As a user, I want to see some data when I do not have any appointments

##  Questions (If any):

### CODE REVIEW

File: src/components/DashboardNav.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        height: auto;
+        border-radius: 50%;
+      }
+    }
+
+    .profileMenu:hover {
+      cursor: pointer;
+    }
+  }
+`;
```
Comment (BenjaminGrabow) [inline src/components/DashboardNav.js]: we need to delete this file, we don't use it in our application anymore

**Comment by domeccleston** [comment]
The placeholder image looks good but we have some problems here, the landing page and login form styles are broken 

Screenshot from 2020-01-16 14-24-17

Screenshot from 2020-01-16 14-24-21

**Comment by Funmi7** [comment]
> The placeholder image looks good but we have some problems here, the landing page and login form styles are broken
> 
> Screenshot from 2020-01-16 14-24-17
> 
> Screenshot from 2020-01-16 14-24-21

That's weird, didn't touch those components.

**Comment by domeccleston** [comment]
Was due to the ant design css , all good

**Comment by Funmi7** [comment]
[As a user, I want to see some data when I do not have any appointments](https://trello.com/c/XG1H5iAy/92-as-a-user-i-want-to-see-some-data-when-i-do-not-have-any-appointments)

**Pull request merged**